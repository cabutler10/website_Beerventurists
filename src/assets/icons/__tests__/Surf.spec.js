import React from 'react'
import { render } from 'react-testing-library'
import SurfIcon from '../Surf'

describe('SurfIcon', () => {
  it('renders correctly', () => {
    const component = render(<SurfIcon />)
    expect(component).toMatchSnapshot()
  })
})
