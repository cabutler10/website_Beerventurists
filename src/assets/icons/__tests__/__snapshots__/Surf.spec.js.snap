// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurfIcon renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        aria-labelledby="title"
        height="32"
        viewBox="0 0 512 512"
        width="32"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title
          id="title_twitter"
        >
          Surf Icon
        </title>
        <path
          d="M495.487,381.376c-114.617,0.044-191.828-40.025-211.837-109.907c-14.51-50.677,5.895-121.064,58.993-123.655 c57.287,3.776,58.599,67.743,56.683,117.647c-0.707,18.569-1.131,29.775,2.686,37.991c4.361,9.352,14.072,13.86,24.839,11.538 c24.871-5.438,71.035-59.757,71.361-122.848c0.479-94.231-76.146-160.736-186.363-161.731c-0.881-0.006-1.741-0.011-2.631-0.011 C157.349,30.401,2.947,148.965,0,376.977v87.918c0,9.225,7.479,16.704,16.704,16.704h478.592c9.226,0,16.704-7.479,16.704-16.704 v-66.834C512,388.916,504.633,381.372,495.487,381.376z M160.407,448.193c-47.302-81.079-30.474-176.417,29.141-239.361 c-42.78,82.395-20.719,187.048,50.753,239.361H160.407z"
          fill="currentColor"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      aria-labelledby="title"
      height="32"
      viewBox="0 0 512 512"
      width="32"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title
        id="title_twitter"
      >
        Surf Icon
      </title>
      <path
        d="M495.487,381.376c-114.617,0.044-191.828-40.025-211.837-109.907c-14.51-50.677,5.895-121.064,58.993-123.655 c57.287,3.776,58.599,67.743,56.683,117.647c-0.707,18.569-1.131,29.775,2.686,37.991c4.361,9.352,14.072,13.86,24.839,11.538 c24.871-5.438,71.035-59.757,71.361-122.848c0.479-94.231-76.146-160.736-186.363-161.731c-0.881-0.006-1.741-0.011-2.631-0.011 C157.349,30.401,2.947,148.965,0,376.977v87.918c0,9.225,7.479,16.704,16.704,16.704h478.592c9.226,0,16.704-7.479,16.704-16.704 v-66.834C512,388.916,504.633,381.372,495.487,381.376z M160.407,448.193c-47.302-81.079-30.474-176.417,29.141-239.361 c-42.78,82.395-20.719,187.048,50.753,239.361H160.407z"
        fill="currentColor"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
